<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Вход в аккаунт</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon/icon_3.png</normaloff>icon/icon_3.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Bahnschrift Light</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Введите имя пользователя</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="name_line">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Bahnschrift Light</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	background-color: white;
	color: black;
	border-radius: 4px;
	border: 1px solid gray;
} 
QLineEdit:focus {
	background-color: white;
	color: black;
	border-radius: 4px;
	border: 1.5px solid gray;
}
QLineEdit:disabled {
	border: 1px solid gray;
	border-radius: 4px;
	background-color: rgb(200, 200, 200);
	color:  rgb(100, 100, 100);
}</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Bahnschrift Light</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Введите пароль</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="password_line">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Bahnschrift Light</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	background-color: white;
	color: black;
	border-radius: 4px;
	border: 1px solid gray;
} 
QLineEdit:focus {
	background-color: white;
	color: black;
	border-radius: 4px;
	border: 1.5px solid gray;
}
QLineEdit:disabled {
	border: 1px solid gray;
	border-radius: 4px;
	background-color: rgb(200, 200, 200);
	color:  rgb(100, 100, 100);
}</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QPushButton" name="button">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>20</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift Light</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: white;
	color: black;
	border-radius: 4px;
	border: 1px solid gray;
}
QPushButton:hover, QPushButton:focus {
	background-color: white;
	color: black;
	border-radius: 4px;
	border: 2px solid gray;
}
QPushButton:disabled {
	border: 1px solid gray;
	border-radius: 4px;
	background-color: rgb(200, 200, 200);
	color:  rgb(100, 100, 100);
}</string>
     </property>
     <property name="text">
      <string>Войти</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="status_label">
     <property name="font">
      <font>
       <family>Bahnschrift Light</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>name_line</tabstop>
  <tabstop>password_line</tabstop>
  <tabstop>button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
